src/MissionManager/StructureScanComplexItem.cc

```diff
@@ -532,8 +532,10 @@ void StructureScanComplexItem::_rebuildFlightPolygon(void)
     int savedEntryVertex = _entryVertex;
     _entryVertex = 0;
 
-    _flightPolygon = _structurePolygon;
-    _flightPolygon.offset(_cameraCalc.distanceToSurface()->rawValue().toDouble());
+    if(_structurePolygon.isValid()){
```

Comment:
Just being a bit picky, but please here we should have spaces in the "if" line:

if (_structurePolygon.isValid()) {

Thanks!

Reply:
Thank you for the feedback! I've addressed the code style issues in the pull request. Please let me know if there's anything else that needs improvement..